[12.0][ADD] website_snippet_carousel_product (#369)
OCA/e-commerce: website_snippet_carousel_product/readme/USAGE.rst

#. Go to website (frontend)
#. Press 'Edit'

> #. Drag & Drop the 'Carousel Products' snippet

Reviewer: ```suggestion
#. Drag & Drop the 'Products Carousel' snippet
```

> #. Drag & Drop the 'Products Carousel' snippet

Reviewer 2: In UI the used word is block, isn't it?

```suggestion
#. Drag & Drop the 'Products Carousel' block
```

#. Write the domain or leave it empty to show all published products
#. Save

> Theming

Reviewer 2: I would put this more in the configuration section.

~~~~~~~

> This snippet uses 'native' bootstrap carousel and the 'website_sale.products_item' qweb template

Reviewer: ```suggestion

This snippet uses 'native' bootstrap carousel and the ``website_sale.products_item`` qweb template
```
